app/views/recipes/_form.html.erb

```diff
@@ -0,0 +1,43 @@
+<br>
+<%= bootstrap_form_with(model: recipe) do |form| %>
+  <% if recipe.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(recipe.errors.count, "error") %> prohibited this recipe from being saved:</h2>
+
+      <ul>
+        <% recipe.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name, class: "input-default" %>
+  </div>
+
+  <div>
+    <%= form.label :preparation_time, style: "display: block" %>
+    <%= form.number_field :preparation_time, class: "input-default" %>
+  </div>
+
+  <div>
+    <%= form.label :cooking_time, style: "display: block" %>
+    <%= form.number_field :cooking_time, class: "input-default" %>
+  </div>
+
+  <div>
+    <%= form.label :description, style: "display: block" %>
+    <%= form.text_area :description, class: "input-default" %>
+  </div>
+
+  <div class="row checkbox">
+    <%= form.check_box :public %>
+    <%= form.label :public, style: "display: block" %>
+  </div>
```

Comment:
The labels for the add recipe form inputs are also repeated twice: 
![image](https://github.com/IsmailChangezi/recipe-rails/assets/93520178/48e31f92-1dca-4603-895f-359fd40229a8)
Kindly fix this issue.